TaskLayer/NeoSearchTask/NeoSearchTask.cs

```diff
@@ -25,13 +25,8 @@ public NeoSearchTask() : base(MyTask.Neo)
         {
             NeoParameters = new NeoParameters();
 
-            IDigestionParams tempDigParams = new DigestionParams
-            {
-                MinPeptideLength = 8,
-                MaxPeptideLength = 13,
-                Protease = GlobalVariables.ProteaseDictionary["non-specific"],
-                MaxMissedCleavages = 12
-            };
+            DigestionParams tempDigParams = new DigestionParams(GlobalVariables.ProteaseDictionary["non-specific"], 12, 8, 13, 1024, InitiatorMethionineBehavior.Variable, 2, false, TerminusType.N);
```

Comment:
this is too many parameters specified I think, try to make very minimal changes to the previous structure